src/Cake.Prca.Tests/FakeCodeAnalysisProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Cake.Prca.Tests
+{
+    using System.Collections.Generic;
+    using Core.Diagnostics;
+    using Prca.Issues;
+
+    public class FakeCodeAnalysisProvider : CodeAnalysisProvider
+    {
+        private readonly List<ICodeAnalysisIssue> issues = new List<ICodeAnalysisIssue>();
+
+        public FakeCodeAnalysisProvider(ICakeLog log) 
+            : base(log)
+        {
+        }
+
+        public FakeCodeAnalysisProvider(ICakeLog log, IEnumerable<ICodeAnalysisIssue> issues)
+            : base (log)
```

Comment:
StyleCop has a lot of rules which make absolutely sense even for test assemblies, like consistent encoding etc. I think test code should be treated like normal code, and we should demand the same level of maintainability and readability also there.